sdk/identity/identity/samples-dev/clientSecretCredential.ts

```diff
@@ -1,6 +1,10 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
+/**
+ * @summary Authenticates with a client and a app registration's secret.
```

Comment:
could you update this summary as well?

Reply:
Oh now I think I should have said `Authenticates with a client Id and an app registration’s secret`. I’ll update it!

Reply:
How does this look? https://github.com/Azure/azure-sdk-for-js/pull/19085/commits/9f525fcbedc230acf43fc50b9e676233a45340c2

Reply:
Looks good, thanks!